walker/src/pages/CoOpHome.tsx

```diff
@@ -101,32 +120,34 @@ function CoOpHome({ user }: { user: IUser }): JSX.Element {
                         Pending Walker Calls
                     </p>
                     <ul>
-                        {calls.map((call: ICall, index: number) => (
-                            <li
-                                key={index}
-                                style={{
-                                    display: "block",
-                                    whiteSpace: "nowrap",
-                                    marginBottom: "5px",
-                                }}
-                            >
-                                <div>
-                                    <strong>{call.requester?.pet_name}</strong>{" "}
-                                    - {call.activity} -{" "}
-                                    {new Date(call.date).toLocaleString()}
-                                
-                                    {" "}
-                                    "{call.details}"
-                                
-                                <button
-                                    className="btn"
-                                    style={{ display: "inline-block" }}
-                                >
-                                    Accept Call
-                                </button>
-                                </div>
-                            </li>
-                        ))}
+
+<div style={{ display: "flex", flexDirection: "column" }}>
+  <div style={{ display: "flex", flexDirection: "row", fontWeight: "bold" }}>
+    <div style={{ flex: 1 }}>Pet Name</div>
+    <div style={{ flex: 1 }}>Activity</div>
+    <div style={{ flex: 1 }}>Time</div>
+    <div style={{ flex: 2 }}>Details</div>
+    <div style={{ flex: 1 }}>Status</div>
+  </div>
+  {calls.map((call: any, index: number) => (
+    <div key={index} style={{ display: "flex", flexDirection: "row", marginBottom: "5px" }}>
+      <div style={{ flex: 1 }}>{call.requester?.pet_name}</div>
+      <div style={{ flex: 1 }}>{call.activity}</div>
+      <div style={{ flex: 1 }}>{new Date(call.date).toLocaleString()}</div>
+      <div style={{ flex: 2 }}>{call.details}</div>
+      <div style={{ flex: 1 }}>
+        {(acceptedCalls.has(call._id))? (
```

Comment:
since `acceptedCalls` gets set with useState, it'll get reset everytime you refresh the page, & everything will show up as unaccepted whenever you refresh or navigate from another page.